### PR TITLE
Goalgetters/support page

### PR DESCRIPTION
**Pull request by missdobis**
Support page css styling and responsiveness

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,8 +21,7 @@
     "gatsby-plugin-sass": "^2.3.14",
     "node-sass": "^4.14.1",
     "react": "^16.12.0",
-    "react-dom": "^16.12.0",
-    "react-icons": "^3.11.0"
```
Comment (kingflamez) [inline package.json:25]: Are another group not using the react icons? Why remove it?

**Review by kingflamez — CHANGES_REQUESTED** [review]
Perfect please, check the react-icons before i merge

File: package.json (hunk 1 above)
Comment (joshtom) [reply package.json:25]: I installed it to use Icons then felt like I don't need it again. That's why I uninstalled it. Would have to remove it from the package.json then.

Comment (kingflamez) [reply package.json:25]: Oh, no it has been removed, just wanted to confirm other groups are not using it

**Comment by kingflamez** [comment]
Screenshot 2020-10-05 at 11 10 14

Please fix this lint issue

**Comment by kingflamez** [comment]
Screenshot 2020-10-05 at 11 13 42

The padding is not consistent with the design

**Pull request merged**